### ISSUE TITLE
Возможность задать seed игре

### ISSUE THREAD

**Issue by Mur4ik** [sat2707/aicups#30]
Когда у нас появится локалраннер, оч хотелось бы иметь такую возможность для локальных извращений

**Comment by sat2707** [comment]
Да, безусловно сделаем такую возможность. Кроме того, показывать seed у сгенеренной на сайте партии тоже будет полезно.
